udata/tests/dataset/test_dataset_model.py

```diff
@@ -89,13 +89,6 @@ def test_update_resource_missing_checksum_type(self):
         with pytest.raises(db.ValidationError):
             dataset.update_resource(resource)
 
-    def test_last_update_with_resource(self):
```

Comment:
I think we should keep this test, but use resource.created_at instead ;)

We could add a case with an harvest date being the max